modules/ROOT/pages/backup-restore/restore-dump.adoc

```diff
@@ -140,6 +145,20 @@ You can load a dump from a local directory using the following command:
 bin/neo4j-admin database load --from-path=/full-path/data/dumps <database> --overwrite-destination=true
 ----
 
+Starting from Neo4j 5.20, you can use the same command to load the database from its full backup artifact:
+
+[source,shell, role="nocopy"]
+----
+bin/neo4j-admin database load --from-path=/full-path/to/backups <database> --overwrite-destination=true
```

Comment:
I wonder if the path shouldn't point to the artifact as it does in the `neo4j-admin database restore` command.

Reply:
tbh, I think it's redundant in this case